### ISSUE TITLE
about remote management on Rainbow miner clients

### ISSUE THREAD

**Issue by GFreedomX** [RainbowMiner/RainbowMiner#2728]
Hi guys,
Quite new to rainbow miner.
i have install / setup the follow
1)Linux base rainbow miner as server
2)Windows base with rainbow miner installed as client

Currently I am not able to see the windows client of rainbow miner in my linux rainbow miner server.

but on the ranbow miner I see the client connect. 
![image](https://github.com/RainbowMiner/RainbowMiner/assets/48503005/1f832f7e-04c9-4885-bb44-fdae419b70ea)

but on the dashboard I don't see it.
been going thru all the thing I can google but nothing point to it.

so I was wondering if this need to enable in order for it to show up
![image](https://github.com/RainbowMiner/RainbowMiner/assets/48503005/e5950037-4404-43b3-a1e4-dabd23152bda)

both the server and client is using port 4000

as help /suggest will be great.

Is there also a discord for rainbow miner?

**Comment by GFreedomX** [comment]
Sorry about this. after i applied the reporting / monitoring part it works.

**Comment by whitindllc** [comment]
yeah you need to enable reporting and put your api/status key there
